api/portainer.go

```diff
@@ -388,6 +388,9 @@ type (
 		LastCheckInDate int64
 		// QueryDate of each query with the endpoints list
 		QueryDate int64
+		// Heartbeat indicates the heartbeat status of an edge environment
+		Heartbeat bool `json:"Heartbeat" example:"true"`
```

Comment:
May I suggest instead of adding the property to the endpoint object, just using a struct that extends Endpoint for `endpoint_list`? otherwise it's confusing for other apis that return the endpoint without a valid heartbeat